### ISSUE TITLE
Crash when clicking "Change Database"

### ISSUE THREAD

**Issue by yschroeder** [PhilippC/keepass2android#1055]
I managed to crash Keepass2Android realiably by pressing the `Change Database` button.

I tried to add a new webdav connection. After creating the connection with a `https://` URL and running into errors, I tried to change the protocol to `webdavs://` by editing the database connection. Right after Keepass crashed. Now it crashes every time I press `Change Database` in the main screen.

Stack trace is something like this:
```
android.runtime.JavaProxyThrowable: keepass2android.NoFileStorageFoundException:
Unknown protocol webdavs://username:password@url at
keepass2android.Kp2aApp.GetCloudFileStorage (KeePassLib.Serialization.IOConnectionInfo iocInfo)
(...)
```

This is only the first part of the stack trace (as I did not want to type everything from my screen...).

Basically, I cannot change databases anymore (or remove the faulty one) as it keeps crashing.

I am running version 1.07b-r0 from Play Store.

**Comment by PhilippC** [comment]
if you are certain that you won't lose any data, it's probably best to delete the app settings and start from scratch. Sorry about that.

**Comment by yschroeder** [comment]
Ok, that worked. Thank you for this great program!
